docs/migrating11to17.md

```diff
@@ -0,0 +1,33 @@
+<!--
+* Copyright (c) 2017, 2022 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+
+# Migrating from Java 11 to Java 17
+
+Support for OpenJDK 17 was added in OpenJ9 version 0.29.1.
+
+The following new OpenJ9 changes apply when OpenJ9 is built with Java SE 17 class libraries. This information exists elsewhere in the documentation but is summarized here for convenience:
+
+- Support for JDK enhancement proposals (JEPs) as described in [What's new in version 0.29.1](version0.29.1.md).

```

Comment:
This should also include the JEPs added in Java 12 - 16.

Reply:
@pshipton - We have JEP list only for Java 15, 16 and 17, not any earlier releases. We can add the JEP list to the What's new topic of the earlier releases, if you could provide the same. 

We have a JEP list for Java 9, 10, and 11 in https://www.ibm.com/docs/en/semeru-runtime-ce-z/11?topic=new-first-release. Can this list be used here?

Reply:
> We have a JEP list for Java 9, 10, and 11. Can this list be used here?

Sure.

Reply:
Raw list https://openjdk.org/projects/jdk/15

339: 	Edwards-Curve Digital Signature Algorithm (EdDSA) - OpenJDK
360: 	Sealed Classes (Preview)
371: 	Hidden Classes
372: 	Remove the Nashorn JavaScript Engine - OpenJDK
373: 	Reimplement the Legacy DatagramSocket API - OpenJDK
375: 	Pattern Matching for instanceof (Second Preview) - OpenJDK
378: 	Text Blocks - OpenJDK
383: 	Foreign-Memory Access API (Second Incubator)
384: 	Records (Second Preview)
385: 	Deprecate RMI Activation for Removal - OpenJDK

Reply:
We don't really need to distinguish between what required changes to OpenJ9 and what did not.